x/rollappparams/types/params.go

```diff
@@ -0,0 +1,145 @@
+package types
+
+import (
+	"fmt"
+	"regexp"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/dymensionxyz/dymint/da/registry"
+	"github.com/dymensionxyz/gerr-cosmos/gerrc"
+)
+
+const (
+	// current supported DA
+	DefaultDA = "celestia"
+	// commit used for the rollapp binary. it must be overwritten in Makefile.
+	DefaultCommit = "74fad6a00713cba62352c2451c6b7ab73571c515"
+	// default max gas accepted per block. limited to 400M.
+	DefaultBlockMaxGas = 400000000
+	// default max block size accepted (equivalent to block max size it can fit into a celestia blob).
+	DefaultBlockMaxSize = 500000
+	// default minimum block size. not specific reason to set it to 100K, but we need to avoid no transactions can be included in a block.
+	MinBlockMaxSize = 100000
+	// default minimum value for max gas used in a block. set to 10M to avoid using too small values that limit performance and avoid no transactions can be included in a block.
+	MinBlockMaxGas = 10000000
+	// length of the commit string.
+	CommitLength = 40
+)
+
+// Parameter store keys.
+var (
+	KeyDa           = []byte("da")
+	KeyCommit       = []byte("commit")
+	KeyBlockMaxGas  = []byte("blockmaxgas")
+	KeyBlockMaxSize = []byte("blockmaxsize")
+	CommitRegExp    = regexp.MustCompile(`^[a-z0-9]*$`)
+)
+
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// NewParams creates a new Params object
+func NewParams(
+	da string,
+	commit string,
+	blockMaxGas uint32,
+	blockMaxSize uint32,
+) Params {
+	return Params{
+		Da:           da,
+		Commit:       commit,
+		Blockmaxgas:  blockMaxGas,
+		Blockmaxsize: blockMaxSize,
+	}
+}
+
+// DefaultParams returns default x/rollappparams module parameters.
+func DefaultParams() Params {
+	return Params{
+		Da:           DefaultDA,
+		Commit:       DefaultCommit,
+		Blockmaxgas:  uint32(DefaultBlockMaxGas),
+		Blockmaxsize: uint32(DefaultBlockMaxSize),
+	}
+}
+
+func (p Params) Validate() error {
+	err := ValidateDa(p.Da)
+	if err != nil {
+		return err
+	}
+	err = ValidateCommit(p.Commit)
+	if err != nil {
+		return err
+	}
+	err = ValidateBlockMaxGas(p.Blockmaxgas)
+	if err != nil {
+		return err
+	}
+	err = ValidateBlockMaxSize(p.Blockmaxsize)
+	if err != nil {
+		return err
+	}
```

Comment:
need to validate min?

Reply:
there is only maxblocksize limit, which cannot be lower that MinBlockMaxSize that im already validating. or maybe i didnt get your comment...